mods/ra/weapons/explosions.yaml

```diff
@@ -142,15 +142,41 @@ CivPanicExplosion:
 		Delay: 1
 
 BarrelExplode:
+	Warhead@Cluster: FireCluster
+		Weapon: BarrelCluster
+		Dimensions: 3,3
+		Footprint: _X_ X_X _X_
+	Warhead@1Dam: SpreadDamage
+		Spread: 325
+		Falloff: 1000, 368, 135, 50, 18, 7, 0
+		Delay: 5
+		ValidTargets: GroundActor, Trees
+		Versus:
+			None: 120
+			Light: 50
+			Wood: 20
+			Concrete: 10
+	Warhead@2Eff: CreateEffect
+		Explosions: building
+		ImpactSounds: kaboom25.aud
+		Delay: 5
+	Warhead@Smu: LeaveSmudge
+		SmudgeType: Crater
+		Delay: 1
```

Comment:
This makes the center smudge appear before the center damage and explosion?

Also, personally I find it would look better if the center explosion (so basically, the non-cluster warheads here on BarrelExplode) weren't delayed at all.